examples/fluids/src/cloptions.c

```diff
@@ -131,9 +131,6 @@ PetscErrorCode ProcessCommandLineOptions(MPI_Comm comm, AppCtx app_ctx, SimpleBC
   // Inflow BCs
   bc->num_inflow = 16;
   PetscCall(PetscOptionsIntArray("-bc_inflow", "Face IDs to apply inflow BC", NULL, bc->inflows, &bc->num_inflow, NULL));
-  // Outflow BCs
-  bc->num_outflow = 16;
-  PetscCall(PetscOptionsIntArray("-bc_outflow", "Face IDs to apply outflow BC", NULL, bc->outflows, &bc->num_outflow, NULL));
```

Comment:
Why delete outflow boundaries? I know we like to use `bc_freestream` for convenience and to avoid reflections, but some tests deliberately use outflow.

Reply:
Alright, I will return the outflow boundaries.

Reply:
I'm working in your branch and will push shortly with everything working (and these outflows restored). But there's a workflow issue here.
1. You should run tests with every commit. `make test search=fluids BACKENDS=/cpu/self` runs in half a minute or so and would show that deleting the outflows broke a bunch of examples that used outflows. Those examples would need to be updated in the same commit. (I happen to think it's useful to keep around.)
2. The documentation still talks about outflow conditions. We don't make commits that break documentation so if you really were removing it, you'd need to update the documentation.
3. A major functional change like that needs to be explained in a commit message, but the commit message didn't state that.

Reply:
Ok, I will wait for your push. I will keep in mind to run test before any commit.